syncstorage/tests/functional/test_storage.py

```diff
@@ -1286,6 +1287,49 @@ def test_pagination_with_newer_and_sort_by_oldest(self):
                 self.assertEquals(sorted(int(item['id']) for item in items),
                                   range(start + 1, NUM_ITEMS))
 
+    def test_transactions(self):
+        endpoint = self.root + '/storage/col2'
+        bso1 = {'id': '12', 'payload': 'elegance'}
+        bso2 = {'id': '13', 'payload': 'slovenly'}
+        bsos = [bso1, bso2]
+        self.app.post_json(endpoint, bsos)
+
+        bso3 = {'id': 'a', 'payload': 'internal'}
+        bso4 = {'id': 'b', 'payload': 'pancreas'}
+        resp = self.app.post_json(endpoint + '?batch=true', [bso3, bso4])
+        batch = resp.json["batch"]
+        # This is sooooo going to break when it's run at just the wrong time
+        # and will result in a seriously confused person at the keyboard.
+        guess = float(resp.headers["X-Weave-Timestamp"])
+        self.assertEqual(batch / 1000, math.floor(guess))
```

Comment:
Since the value of batch is defined to be opaque, we probably don't need to check that it has any particular value, but rather just check that it's an appropriate-looking string.